packages/frameworks-astro/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@auth/astro",
+  "description": "Authentication for Astro.",
+  "version": "0.1.0",
+  "type": "module",
+  "files": [
+    "api",
+    "src",
+    "middleware.ts",
+    "module.d.ts"
```

```suggestion
   "*.d.ts*",
    "*.js",
    "lib",
    "providers",
    "src"
```

more future-proof, requires less changes. let's follow the same structure as other integrations

---

This makes sense, although the Astro package will be unique in the fact it doesn't require compiling, so we can just track `*.ts`

---

Cleaned up the files we do include